priv/perf/apps/load_test/lib/child_chain/exit.ex

```diff
@@ -63,21 +63,18 @@ defmodule LoadTest.ChildChain.Exit do
   Retries until the exit data of a utxo is found.
   """
   @spec wait_for_exit_data(Utxo.t(), pos_integer()) :: any()
-  def wait_for_exit_data(utxo_pos, counter \\ 10)
-  def wait_for_exit_data(_, 0), do: :error
+  def wait_for_exit_data(utxo_pos, timeout \\ 100_000) do
+    func = fn ->
+      data = get_exit_data(utxo_pos)
 
-  def wait_for_exit_data(utxo_pos, counter) do
-    data = get_exit_data(utxo_pos)
+      if not is_nil(data.proof) do
```

Comment:
Should data.proof ever be `nil`?

Reply:
It can be nil. Because its key is set directly from response data without checking the result